plugin.video.orftvthek/resources/language/German/strings.po

```diff
@@ -0,0 +1,234 @@
+# XBMC Media Center language file
```

Comment:
change this to `plugin.video.orftvthek/resources/language/resource.language.de_de/strings.po`

Reply:
also done